fuzzy/test/test_fuzzy_python.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this is a Windows-ism, and in general we want ASCII... I think I'll have to fix this on my end after the merge

Reply:
Also, the file should start with `#!/usr/bin/env python`